definitions/npm/formik_v0.11.x/flow_v0.53.x-/formik_v0.11.x.js

```diff
@@ -1,3 +1,6 @@
+import { ComponentType } from "react";
+import { FormikProps } from "formik";
```

Comment:
I don't think this currently works, see discussion [here](https://github.com/flowtype/flow-typed/issues/1857).

Reply:
this is wrong I guess